### ISSUE TITLE
When reducing the training set characters, an error is reported:

### ISSUE THREAD

**Issue by jingmingtao** [clovaai/mxfont#32]
When reducing the training set characters, an error is reported:  File "/home/jmt/MXfont/mxfont-main/trainer/trainer_utils.py", line 101, in expert_assign r_in, c_in = linear_sum_assignment(prob_in)
ValueError: matrix contains invalid numeric entrie. 

There should be a free data matrix, but increasing the batchsize will not cause an error, but I don't want to increase the batchsize. May I ask if there is any solution available

**Comment by jingmingtao** [comment]
My data is fine, it only uses 556 characters. May I ask what is causing this? I have been locating it for a long time and cannot solve it. Can you provide the following assistance

**Comment by 8uos** [comment]
It seems that the training is unstable. "ValueError: matrix contains invalid numeric entrie." means that the input contains NaN, which can caused by various factors. I cannot know what the reason exactly is, but I guess using less characters than our setting may cause unstable training.

**Comment by jingmingtao** [comment]
Yes, I used 556 characters for style and 6763 characters for content to train. You guessed it right.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Song ***@***.***&gt;; 
发送时间: 2023年11月21日(星期二) 中午11:44
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [clovaai/mxfont] When reducing the training set characters, an error is reported: (Issue #32)





 
It seems that the training is unstable. "ValueError: matrix contains invalid numeric entrie." means that the input contains NaN, which can caused by various factors. I cannot know what the reason exactly is, but I guess using less characters than our setting may cause unstable training.
